app/views/includes/degateway-banner.njk

```diff
@@ -1,10 +1,11 @@
-<div class="govuk-phase-banner">
-  <p class="govuk-phase-banner__content">
-    <strong class="govuk-tag govuk-phase-banner__content__tag govuk-tag--orange">
-      Experimental
-    </strong>
-    <span class="govuk-phase-banner__text">
-      Your account is opted in to the experimental degateway view – <a class="govuk-link" href="{{ routes.user.profile.degateway }}">opt out</a>.
-    </span>
-  </p>
-</div>
+{% if isDegatewayed %}
```

Comment:
it's clearer from a developers perspective if the logic block that conditionally renders this include is in the template it's imported in rather than the the include itself